### ISSUE TITLE
AutoPrefixer not working?

### ISSUE THREAD

**Issue by SachaG** [fourseven/meteor-scss#77]
For some reason my SCSS isn't getting prefixed anymore. I just noticed it now so it's probably a fairly recent change. I'm not sure if it's due to the app ([Telescope](https://github.com/TelescopeJS/Telescope/)) or the package (I'm on `2.0.1_5`) though. 

Here's my `scss.json`:

```
{
  "enableAutoprefixer": true,
  "outputStyle": "compressed",
  "sourceComments": true,
  "sourceMap": true
}
```

**Comment by SachaG** [comment]
So I can confirm that downgrading to version `1.2.3` re-enabled autoprefixing.

**Comment by fourseven** [comment]
Thanks for reporting it @SachaG - I'll look into it later on this afternoon (~6h)

**Comment by fourseven** [comment]
@SachaG - is the autoprefixing broken or is it just related to sourcemaps? I'm trying it on my project and it's working but not in the sourcemapped scss file - it is in the output css for me.

If it's broken, can you give me an example property please?

**Comment by SachaG** [comment]
No, I meant the actual autoprefixing. `display:flex` for example.

**Comment by SachaG** [comment]
(The problem is happening on Telescope's [devel](https://github.com/TelescopeJS/Telescope/tree/devel) branch if you want to try and run in)

**Comment by fourseven** [comment]
Thanks I'm cloning now

**Comment by fourseven** [comment]
it broke with #68 - the project root fallback isn't working. I need to cut a 2.1.1 release, so I'll include the fix in that.

**Comment by SachaG** [comment]
Oh, makes sense. Thanks for the quick response :)

**Comment by fourseven** [comment]
All published – `fourseven:scss@2.1.1` – happy to help, I loved discover meteor!

**Comment by SachaG** [comment]
Awesome :)
